Python/specialize.c

```diff
@@ -215,6 +215,7 @@ print_gc_stats(FILE *out, GCStats *stats)
     }
 }
 
+#ifdef _Py_TIER2
```

Comment:
Are the new ifdefs in this file related to the rest of the PR?

Reply:
They are just to get the stats build to work. But I'll need to add these fixes to main to get comparative stats, so they'll get removed from this PR.